nixos/modules/system/boot/systemd.nix

```diff
@@ -1212,6 +1212,23 @@ in
     boot.kernel.sysctl."kernel.pid_max" = mkIf pkgs.stdenv.is64bit (lib.mkDefault 4194304);
 
     boot.kernelParams = optional (!cfg.enableUnifiedCgroupHierarchy) "systemd.unified_cgroup_hierarchy=0";
+
+    services.logrotate.paths = {
+      "/var/log/btmp" = {
```

Comment:
Same as above please.

Reply:
Replying here for the name-as-path or separate-path as I can't split the other conversation:
I think it's better to use path directly as the name as there's only one canonical way of doing it for single files, so it's easier to override. I'm thinking of it like environment.etc.
For wildcards it's arguable as there would be many ways of doing it, I'm not averse to naming it nginx and setting the paths separately. I'll only update nginx for now, `services.logrotate.paths."/var/log/btmp"` and wtmp already existed and some people might already be using them (although it's not in any release yet, so if you feel strongly about it there's still time to rename it)

Reply:
I have a preference to use an application name instead of the path because the path can be anything, including a multi line string. This can be painful for users to customize or disable.

Since `wtmp` is a single file I think it is reasonable to keep the name the same as the path.

So the way you have it at this point seems perfect to me 👍